routes/index.js

```diff
@@ -1,22 +1,44 @@
 const express = require("express");
 const router = express.Router();
+const Post = require("../models/post");
 
 /* GET home page. */
-router.get("/", function (req, res, next) {            /* Landing or HomePage */
-
+router.get("/", function (req, res, next) {
+	res.render("index", { title: "Express" });
 });
 
-router.get("/posts", function(req, res, next){           /* display the lists of post in db */
-
+/* create post */
+router.post("/post", function (req, res) {
+	var got = {
+		given_name: req.body.username,
+		given_location: req.body.location,
+		given_image: req.body.image,
+		given_description: req.body.description,
+	};
+	/*console.log(got,req.body)*/
+	var Details = new Post({
+		username: got.given_name,
+		location: got.given_location,
+		image: got.given_image,
+		description: got.given_description,
+	});
+	Details.save();
+	res.redirect("/posts");
```

Comment:
you need to send the Post just created, please make that change using res.send, also add code to catch the error if any and send appropriate message